internal/cmd/org.go

```diff
@@ -21,6 +21,51 @@ func init() {
 	membersCmd.AddCommand(membersRemoveCmd)
 }
 
+func switchToOrg(client *turso.Client, slug string) error {
+	settings, err := settings.ReadSettings()
+	if err != nil {
+		return err
+	}
+	orgs, err := client.Organizations.List()
+	if err != nil {
+		return err
+	}
+
+	current := settings.Organization()
+	if current == "" {
+		for _, o := range orgs {
+			if o.Type == "personal" {
+				current = o.Slug
+				break
+			}
+		}
+	}
+
+	if current == slug {
+		fmt.Printf("Organization %s already selected\n", internal.Emph(slug))
+		return nil
+	}
+
+	prev := fmt.Sprintf("turso org switch %s", current)
+
+	org, err := findOrgWithSlug(orgs, slug)
+	if err != nil {
+		return err
+	}
+
+	if org.Type == "personal" {
+		slug = ""
+	}
+
+	settings.SetOrganization(slug)
+
+	fmt.Printf("Default organization set to %s.\n", internal.Emph(org.Slug))
```

Comment:
This should be "current" rather than "default".  The CLI doesn't really have a sense of default because you can't override it in any way.  It must be set as the current choice.